objects/app/apps.go

```diff
@@ -210,7 +210,7 @@ func (a *appsCmd) getConfig(c *cli.Context) error {
 
 	val, ok := app.Config[key]
 	if !ok {
-		return fmt.Errorf("Config key does not exist")
```

Comment:
1. To maintain consistency with the equivalent error in functions.go
2. Because Go lang told me to: https://github.com/golang/go/wiki/Errors 

> Error strings should not start with a capital letter because they'll often be prefixed before printing